### PR TITLE
Use numeric error code for function triggering

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Use numeric error code for function triggering results
- Tweak testing params

Related issues: https://github.com/ainblockchain/ain-blockchain/issues/479

### CODE REVIEW

File: db/index.js [hunk 1]
```diff
@@ -1340,6 +1324,15 @@ class DB {
       this.collectFee(auth, timestamp, tx, blockNumber, executionResult);
       this.recordReceipt(auth, tx, blockNumber, executionResult);
     }
+    if (!CommonUtil.isFailedTx(executionResult)) {
+      const heightCheck = this.checkTreeHeightAndSize();
+      if (CommonUtil.isFailedTx(heightCheck)) {
+        if (restoreIfFails) {
+          this.restoreDb();
```
Comment (liayoo) [inline db/index.js:1331]: question) how should we collect fees & record receipts in this case? Maybe leave a todo for this?

Comment (platfowner) [reply db/index.js]: As discussed offline, just reverted the checkpoint change with the plan to simplify the recorded receipts.

**Comment by platfowner** [comment]
PTAL~ @liayoo

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Pull request merged**